src/env.d.ts

```diff
@@ -15,3 +15,9 @@ declare module '*.query.graphql' {
   const value: DocumentNode;
   export = value;
 }
+
+declare module '*.fragment.graphql' {
+  import { DocumentNode } from 'graphql';
+  const value: DocumentNode;
+  export default value;
+}
```

Comment:
this allows for importing the fragments into code. Note the difference in export statement between `*.query.graphql` and `*.fragment.graphql`.